colors/bearded-theme-vivid-black.lua

```diff
@@ -0,0 +1,69 @@
+-- Theme by @xStormyy on GitHub
+-- Original Color Scheme: https://github.com/BeardedBear/bearded-theme/blob/master/themes/bearded-theme-vivid-black.json
+
+
+local style = require "core.style"
+local common = require "core.common"
+
+
+-- base colors
+--[[
+red = "#D62C2C"
+orange = "#FF7135"
+yellow = "#FFB638"
+green = "#42DD76"
+blue = "#28A9FF"
+cyan = "#14E5D4"
+magenta = "#E66DFF"
+white = "#c8c8c8"
+black = "#141417"
+]]--
+
+--[[
+    #////////////////////////////#
+           User Interface
+    #////////////////////////////#
+]]--
+style.background2 = { common.color "#0f0f11" } -- sidebar
+style.background3 = { common.color "#0f0f11" } -- status bar
+style.text = { common.color "#afafafCC" } -- user interface text
+
+style.accent = style.caret -- accent color
```

Comment:
This won't use the `style.caret` you defined later.

Reply:
oh my bad